README.md

```diff
@@ -1448,6 +1451,10 @@ $ git stash push working-directory-path/filename1.ext working-directory-path/fil
 ```sh
 $ git stash save <message>
 ```
+or
+```sh
+$ git push -m <message>
```

Comment:
Do you mean `git stash -m message`?

Reply:
sry, `git stash push -m <message>` i mean,